lib/main_containers.pm

```diff
@@ -155,6 +155,24 @@ sub load_host_tests_helm {
     }
 }
 
+sub load_host_tests_in_k8s {
+    my ($run_args) = @_;
+    my $backends = undef;
+
+    # if (is_sle('15-sp3+')) {
```

Comment:
why this commented ? let's or uncomment it or delete it

Reply:
done